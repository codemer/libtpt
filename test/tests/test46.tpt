@tpt_ignoreblankline\

@macro(cmt) {
	@<@if (1) {
 * indent in a macro
	@<} @else {
 * indent in a macro
	@<} @# comment
 * indent in a macro
}

/*
 * Here's a split up multi-line comment.
@if (1) {
	@<@if (1) {
 * This is indented text.
	@<}
	@<@cmt()\
} @# comment
 * and the last line.
 */

