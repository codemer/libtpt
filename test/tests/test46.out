/*
 * Here's a split up multi-line comment.
 * This is indented text.
 * indent in a macro
 * indent in a macro
 * and the last line.
 */
